### DISCUSSION TITLE
on demand rtsp stream

### DISCUSSION THREAD

**Discussion by sishari** [bluenviron/mediamtx#2835] [Questions]
### Question

Hello,
The mediamtx should only read an RTSP stream from IP Camera when a client (VLC) connects to the mediamtx.
If the client disconnects, the RTSP stream should be stopped.
I use the windows version.

How i must config the yml?

Thanks

**Comment by Taps7734** [comment]
I would look into the "sourceOnDemand: yes" setting.

[a comment by an automated account is not shown]
